backend/app/exporters/serializers/marc_auth.rb

```diff
@@ -360,10 +360,14 @@ def names_person(primary, not_primary, parallel, xml)
     end
   end
 
-  def person_name_subtags(name, xml)
+  def person_name_subtags(name, xml, related = false)
+    if related
```

Comment:
Not a huge deal: the `if related` section is duplicated (in person, fam, corp) so could be refactored to a method.

Reply:
@mark-cooper Updated.